### DISCUSSION TITLE
When I press on the comments icon, take me to that post's comments.

### DISCUSSION THREAD

**Discussion by Nova-Aurora** [Docile-Alligator/Infinity-For-Reddit#902] [Ideas]
Currently the comment icon is only an indicator of how many comments are on a post.
![Screenshot_2020-12-07-11-50-31](https://user-images.githubusercontent.com/62871042/101386658-1b9bd200-3883-11eb-93f6-a7eebf043b46.jpg)
Could we make it a button to open it, scroll past the post itself and go to the comments under it?

**Comment by AndhikaWB** [comment]
+1 would be useful for long post (r/tifu, etc).

**Comment by NomarCub** [comment]
You could also have a configurable layout for comments, the same as posts. You could then set it to compact and the picture won't take up the whole screen. This would be good for other uses too.

**Comment by NomarCub** [comment]
When can we expect a new release with the resolved issues? @Wladefant, @Docile-Alligator

**Comment by Wladefant** [comment]
Hey @NomarCub i dont know why i closed it, sorry, it is not implemented.
